drizzlepac/runastrodriz.py

```diff
@@ -1233,6 +1253,48 @@ def restore_pipeline_default(files):
                 if 'hdrname' in fhdu[('sci', sciext + 1)].header:
                     del fhdu[('sci', sciext + 1)].header['hdrname']
 
+
+def reset_idctab_kw(files, files_flc, logfile=None):
+    """Insure IDCTAB in files are the same as those in RAW files"""
+    trlmsg = "Insuring IDCTAB keywords are up-to-date\n"
+
+    raw_files = [f.replace('_flt', '_raw') for f in files]
+    # determine what IDCTAB should be in FLT files
+    raw_idctab = fits.getval(raw_files[0], 'idctab')
+    raw_wcsname = 'IDC_{}'.format(raw_idctab.split('_')[0].split('$')[1])
+    trlmsg += "IDCTAB from RAW file: {}\n".format(raw_idctab)
+
+    if logfile:
+        # Write message out to temp file and append it to full trailer file
+        _updateTrlFile(logfile, trlmsg)
+    else:
+        print(trlmsg)
+
+    # Now, restore headerlet with this WCSNAME as primary WCS
+    for flt in files+files_flc:
+        flt_idctab = fits.getval(flt, 'idctab')
+        if flt_idctab == raw_idctab:
+            # We don't need to update anything
+            continue
+        newmsg = "Updating IDCTAB {} in {}\n".format(flt_idctab, flt)
+        if logfile:
+            # Write message out to temp file and append it to full trailer file
+            _updateTrlFile(logfile, newmsg)
+        else:
+            print(newmsg)
+
+        # Get info from all hdrlet extensions in file
+        wnames = headerlet.get_headerlet_kw_names('j8mbnlkbq_flc.fits', 'wcsname')
+        hnames = headerlet.get_headerlet_kw_names('j8mbnlkbq_flc.fits', 'hdrname')
```

Comment:
Somehow I do not think you meant to hardcode this explicit filename in this method to get the wname and hnames!

Reply:
Like a broken clock, it would at least be right 1 time during reprocessing.  
Fixed in f6ffd00.